### DISCUSSION TITLE
Dane nie wchodzą do dashboardów

### DISCUSSION THREAD

**Discussion by MrAdrianPl** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#156] [Q&A]
Dzień dobry,
Chciałbym zapytać czy należy napisać jakąś dodatkową automacje, bo dane mimo tego że przychodzą przez integracje to same nie wchodzą do dashbordu energia, trzeba je manualnie usunąć i dodać aby dane się załadowały. 
Nie wiem czy to błąd HAOS, czy gdzieś coś źle skonfigurowałem.
Dodatkowe pytanko czy dane dotyczące kosztów dla danej taryfy są przechowywane w którymś z atrybutów?

**Comment by PiotrMachowski** [comment]
Co masz na myśli że nie wchodzą same? Integracja pobiera je 3 x na dobę


> Dodatkowe pytanko czy dane dotyczące kosztów dla danej taryfy są przechowywane w którymś z atrybutów?

Nie, nie widziałem żeby Tauron udostępniał w ładny sposób koszty dla taryf

**Comment by MrAdrianPl** [comment]
Dzięki za szybką odpowiedź
mam na myśli że dane do dashbordu "energia" nie ładują się przy
odświerzaniu danych z integracji, sprawdzałem dziś i miałem w danych
diagnostycznych dane za 3 dni wstecz a w dashbordzie te dane się nie
doładowywały

niedz., 9 lip 2023, 14:03 użytkownik Piotr Machowski <
***@***.***> napisał:

**Reply by PiotrMachowski** [reply]
Zobacz, czy masz jakieś błędy w logach. Encje się odświeżają?

**Reply by essenemari** [reply]
Potwierdzam. U mnie na pewno było podobnie. Nie do końca odświeżało się od 3 dni mimo kilku planowanych restartów mojego serwera i na pewno IP nie było zablokowane. 
Ale
Przed 2h pociągnęło samo dane.

Na eLicznik jest info że jutro jest przerwa serwisowa na 24h. Moze coś im się wysypało.

**Reply by MrAdrianPl** [reply]
W encjach widzę że jest jednodniowa przerwa ale tak jak @essenemari wyżej miałem przerwę co najmniej 3 dniową, a jako że dopiero co zacząłem używać tej integracji, to uznałem że to jakiś błąd z mojej strony.
Widzę że jednak zrobiłem niepotrzebny szum, dzięki wielkie za odpowiedź i przepraszam za kłopot, zobaczę czy danę będą wchodzić same w następnych dniach.
